hack/get-machine-run-tests.sh

```diff
@@ -0,0 +1,580 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+usage() {
+    local bn=$( basename $0 )
+    echo Usage: $0
+    echo Assumes you have $HOME/origin-aggregated-logging and $HOME/openshift-ansible
```

Comment:
Can we not modify this to be a var other then $HOME.  I, for example, use $HOME/git